packages/sdk/src/types.ts

```diff
@@ -1455,6 +1455,17 @@ export interface SimpleTransferWithMemoPayload extends SimpleTransferPayload {
     memo: DataBlob;
 }
 
+export interface TransferToPublicPayload {
+    data: SecToPubAmountTransferData;
+}
+
+export interface SecToPubAmountTransferData {
+    remainingAmount: string;
+    transferAmount: CcdAmount.Type;
+    index: bigint;
+    proofs: string;
```

Comment:
```suggestion
    proofs: HexString;
```

Signals that we're working with bytes represented as hex here